src/ecredis_server.erl

```diff
@@ -374,6 +376,12 @@ handle_cast({handle_moved, Version, Slot, Node}, State) ->
 handle_cast({remap_cluster, Version}, State) ->
     State2 = remap_cluster_internal(State, Version),
     {noreply, State2};
+handle_cast({ping, Ts, Id}, State) ->
+    case whereis(ejabberd_monitor) of
```

Comment:
We cant have this line. This is os project and others will not have this monitor. We should just reply to the caller.

Reply:
I would have to attach the caller's pid to the ping (`{ping, Ts, Id, From}`?), as there is no way to reply to a cast... alternatively, we could make this a macro, so anyone else using this project can set it to something that works for them (or not at all)